client/app/src/accounts/AccountService.js

```diff
@@ -354,6 +354,12 @@
     };
 
     function verifyMessage(message, publicKey, signature) {
+      //check for hexadecimal, otherwise the signature check would may fail
+      var re = /[0-9A-Fa-f]{6}/g;
+      if(!re.test(publicKey) || !re.test(signature))
+      {
+        return;
```

Comment:
then i would rather return something else from the method.
like `{success:false, error:"error message detailing error"}`